scripts/tasks/copy.ts

```diff
@@ -31,25 +31,49 @@ export function expandSourcePath(pattern) {
 }
 
 export function copy() {
-  let tasks = [];
-  let configPath = path.resolve(process.cwd(), 'config/pre-copy.json');
+  const tasks: TaskFunction[] = [];
+  const configPath = path.resolve(process.cwd(), 'config/pre-copy.json');
 
   if (!fs.existsSync(configPath)) {
     return;
   }
 
-  const config = JSON.parse(fs.readFileSync(configPath, 'utf8'));
+  const config: { copyTo?: Record<string, Array<string> | string> } = JSON.parse(fs.readFileSync(configPath, 'utf8'));
 
   if (config && config.copyTo) {
-    for (let destination in config.copyTo) {
+    for (const destination in config.copyTo) {
       const sources = config.copyTo[destination];
-      destination = path.resolve(process.cwd(), destination);
-      tasks.push(
-        copyTask({
-          paths: sources.map(src => expandSourcePath(src)),
-          dest: destination,
-        }),
-      );
+      const destinationPath = path.resolve(process.cwd(), destination);
+
+      if (Array.isArray(sources)) {
```

Comment:
now there is new api: 
- the old one works as before ( value is a array of string paths/globs)
- the new one is `destination` is the path including new file name + value is only string path (copy + rename)